ToopherDotNet/OAuth/OAuthTools.cs

```diff
@@ -20,6 +20,20 @@ public static class OAuthTools
 
         private static readonly Random _random;
         private static readonly object _randomLock = new object();
+        private static string nonceOverride;
+
+        public static void SetNonceOverride (string nonceOverride)
+        {
+          OAuthTools.nonceOverride = nonceOverride;
+        }
+
+        private static DateTime? dateOverride;
```

Comment:
It was initially used so I could set the dateOverride to `null` for testing. I've since refactored the tests so there's no need to use nullable types. (I've also made the changed in ToopherIframe.)